src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/ReadsPipelineSparkIntegrationTest.java

```diff
@@ -64,6 +65,7 @@ private String getResourceDir(){
                 //Note: these output files were created by running Picard 1.130 and GATK3.46
                 {new PipelineTest(GRCh37Ref2bit_chr2021, hiSeqBam_chr20, dbSNPb37_20, " --joinStrategy BROADCAST", getResourceDir() + EXPECTED_BAM)},
                 {new PipelineTest(GRCh37Ref_2021, hiSeqBam_chr20, dbSNPb37_20, " --joinStrategy SHUFFLE", getResourceDir() + EXPECTED_BAM)},
+                {new PipelineTest(GRCh37Ref2bit_chr2021, hiSeqBam_chr20_queryNameSorted, dbSNPb37_20, " --joinStrategy BROADCAST", getResourceDir() + EXPECTED_BAM)},
```

Comment:
I think this is because there are relatively few splits since each split is around a BGZF block boundary, which occur every 64K or so. The files being used here is 200K, so 4 splits. The code in Hadoop-BAM has some detailed tests for ensuring the pairs aren't split, so it's not the end of the world if we don't have a file that doesn't exhibit read pairs being split.